src/schema.ts

```diff
@@ -67,6 +98,23 @@ export const typeDefs = gql`
     results: [Audio!]
   }
 
+  type PodcastSeriesSummary {
+    id: Int!
+    title: Title!
+    description: Description!
+    supportedLanguages: [String!]
+    episodes: [AudioSummary!]
+    coverPhoto: CoverPhoto!
+  }
```

Comment:
Kunne denne delt definisjon med PodcastSeries? Det er jo kun forskjell på array med episodes. Eller er der ryddigst at det er to separate typer?